Gemfile

```diff
@@ -1,7 +1,7 @@
 source "https://rubygems.org"
 # If you bump the Ruby version, make sure to update the Vagrantfile appropriately
 ruby "2.5.5"
-gem "rails", "4.2.11"
+gem "rails", "5.2.3"
 
 gem "active_model_serializers", "~> 0.9.3"
 gem "aws-sdk", "~> 1.33"
```

Comment:
This blocks upgrade:

```
Bundler could not find compatible versions for gem "nokogiri":
  In Gemfile:
    nokogiri (~> 1.8.1)

    aws-sdk (~> 1.33) was resolved to 1.66.0, which depends on
      aws-sdk-v1 (= 1.66.0) was resolved to 1.66.0, which depends on
        nokogiri (>= 1.4.4)

    axlsx was resolved to 2.0.1, which depends on
      nokogiri (>= 1.4.1)

    fog-aws was resolved to 3.0.0, which depends on
      fog-xml (~> 0.1) was resolved to 0.1.3, which depends on
        nokogiri (>= 1.5.11, < 2.0.0)
```

I'm pretty sure it's possible to switch to just `fog-aws`

Reply:
Resolved by #1071